src/TestSuite/ConnectionHelper.php

```diff
@@ -52,7 +52,7 @@ public function addTestAliases(): void
                 ConnectionManager::alias($connection, $original);
             } else {
                 $test = 'test_' . $connection;
-                ConnectionManager::alias($connection, $test);
+                ConnectionManager::alias($test, $connection);
```

Comment:
So this is saying the `test_` connection must exist for the original connection to work, right?

Should we just skip the non test_ connections since the test_connection must be defined and therefore the alias will eventually be added in the above if condition?

Reply:
> So this is saying the test_ connection must exist for the original connection to work, right?

Yeah, we don't want tests to run in a non test-prefixed connection as that could delete data that is important to the developer. At least with a `test_xyz` connection they will have made an explicit choice on which database will be truncated.